lib/ufe/UsdAttributes.cpp

```diff
@@ -27,9 +27,6 @@
 //		the one in USD.
 // PXR_NAMESPACE_USING_DIRECTIVE
 
-static constexpr char kErrorMsgUnknown[] = "Unknown UFE attribute type encountered";
```

Comment:
Should be
#ifdef UFE_ENABLE_ASSERTS
[lines 30 and 31]
#endif

Reply:
Good catch!